components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsSearchBox/src/EIAnalyticsSearchBox.jsx

```diff
@@ -20,6 +20,8 @@ import Widget from '@wso2-dashboards/widget';
 import Select from 'react-select';
 import './react-select.css';
 
+let pageName;
```

Comment:
Why this variables reside outside of the class?

Reply:
Fixed at b233d5335fc34cc9a03e24a4944cd39d0f6c3677